deps/wazuh_testing/wazuh_testing/qa_ctl/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Change Log
 All notable changes to this tool will be documented in this file.
 
-## [v0.3] - 2021-12-10
+## [v0.3.1] - 2022-03-07
```

Comment:
You have to add new lines related with the new changes, not edit the last version

Reply:
Solved https://github.com/wazuh/wazuh-qa/pull/2640/commits/4623ff4c892f0cd87cc03a1592862750ae6f312b